infrastructure/params-qa.json

```diff
@@ -0,0 +1,14 @@
+{
+  "Parameters": {
+    "ArtifactBucketStack": "sc-environment-codepipeline-s3-cft-qa",
+    "BranchName": "qa",
+    "RepositoryName": "scala-docker",
+    "GitHubOwner": "singlecomm",
+    "GitHubOAuthParam": "/devops/system/github-oauth",
+    "GitHubSecretParam": "github-webhook-secret",
+    "ContainerImage": "aws/codebuild/docker:18.09.0",
+    "ComputeType": "BUILD_GENERAL1_SMALL",
+    "BuildSpecPath": "pipeline/buildspec.yml",
+    "SecretsLambdaArn": "arn:aws:lambda:us-east-1:756142810212:function:secrets-lambda-qa-SecretsLambda-7CC9ANTTDQAK"
```

Comment:
shouldnt the account number here be different for qa?

Reply:
yes